src/state/lists/updater.ts

```diff
@@ -28,6 +28,11 @@ export default function Updater(): null {
     )
   }, [fetchList, isWindowVisible, lists])
 
+  useEffect(() => {
+    if (chainId && [SupportedChainId.OPTIMISM, SupportedChainId.OPTIMISTIC_KOVAN].includes(chainId)) {
+      dispatch(enableList(OPTIMISM_LIST))
+    }
+  }, [chainId, dispatch])
```

Comment:
just add it to the default list of lists

Reply:
hm i see this is for enabling the list rather than adding it to the list of lists